specification/sql/resource-manager/readme.md

```diff
@@ -24,7 +24,7 @@ These are the global settings for the Sql API.
 title: SqlManagementClient
 description: 'The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables you to create, retrieve, update, and delete databases.'
 openapi-type: arm
-tag: package-preview-2023-05
+tag: package-composite-v5
```

Comment:
ARM reviewer, we revert "package-composite-v5" for downstream SDK clients generation. 
It was charged to "package-preview-2023-05" by mistake. thus, all the Avacado failed pipelines could be ignored.